hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/util/Util.java

```diff
@@ -72,6 +73,7 @@
 
     private static final int BUFFER_SIZE = 1 << 15;
     private static final char[] ID_TEMPLATE = "0000-0000-0000-0000".toCharArray();
+    private static final DateTimeFormatter LOCAL_TIME_FORMATTER = DateTimeFormatter.ofPattern("HH:mm:ss.SSS");
```

Comment:
We should use `DateTimeFormatter.ofPattern("HH:mm:ss.SSS", Locale.US);`, even though I'm not sure if it makes a difference in formatting time...

Reply:
no it won't make any difference.. it's just numbers :)